src/components/video/video.jsx

```diff
@@ -1,28 +1,69 @@
+const bindAll = require('lodash.bindall');
+
 const PropTypes = require('prop-types');
 const React = require('react');
 const classNames = require('classnames');
 
 require('./video.scss');
 
-const Video = props => (
-    <div className={classNames('video-player', props.className)}>
-        <iframe
-            allowFullScreen
-            // allowFullScreen is legacy, can we start using allow='fullscreen'?
-            // allow="fullscreen"
-            frameBorder="0" // deprecated attribute
-            height={props.height}
-            scrolling="no" // deprecated attribute
-            src={`https://fast.wistia.net/embed/iframe/${props.videoId}?seo=false&videoFoam=true`}
-            title={props.title}
-            width={props.width}
-        />
-        <script
-            async
-            src="https://fast.wistia.net/assets/external/E-v1.js"
-        />
-    </div>
-);
+class Video extends React.Component {
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            loaded: false
+        };
+    }
+    componentDidMount () {
+        /**
+            uses code snippets from
+            https://github.com/mrdavidjcole/wistia-player-react/blob/master/src/components/wistia_embed.js
+        **/
+        if (!document.getElementById('wistia_script')) {
+            const wistiaScript = document.createElement('script');
+            wistiaScript.id = 'wistia_script';
+            wistiaScript.type = 'text/javascript';
+            wistiaScript.src = 'https://fast.wistia.com/assets/external/E-v1.js';
+            wistiaScript.async = false;
+            document.body.appendChild(wistiaScript);
+        }
+
+        window._wq = window._wq || [];
+        window._wq.push({
+            id: this.props.videoId,
+            onReady: video => {
+                video.bind('secondchange', s => {
```

Comment:
Would be good to have a comment here to describe what we're doing.

Also it looks like this would be firing all the time every second the video is playing.